project/Http4sSite.scala

```diff
@@ -20,6 +20,10 @@ object Http4sSite {
       case (m, (major, minor)) =>
         for ((f, d) <- m) yield (f, s"api/$major.$minor/$d")
     },
+    // TODO: This is a lot of silliness to get a static JSON file onto our site. :(
```

Comment:
This is ridiculous.  Does anybody know why this was necessary?  `target/jekyll` gets populated, but it doesn't make it to `target/site`.